typed-racket-doc/typed-racket/scribblings/reference/types.scrbl

```diff
@@ -354,6 +354,12 @@ corresponding to @racket[trest], where @racket[bound]
 
 @ex[(box "hello world")]
 
+@defform[(Read-Only-Boxof t)]{A read-only @rtech{box} of @racket[t].}
+
```

Comment:
I explained the subtyping and the relationship with other box types, but what do you mean about `const`?